src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1035,7 +1035,7 @@ public function has_passed_retention_time() {
 		// Prevents Pending that is younger than 1 hour to be pruned
 		if (
 			Tribe__Events__Aggregator__Records::$status->pending === $this->post->post_status &&
-			$current > $created + HOUR_IN_SECONDS
+			$current < $created + HOUR_IN_SECONDS
```

Comment:
100% optional - it's not something you introduced in this change - but ideally the `&&` would be at the start of this line, not the end of the last line [(per standards)](http://moderntribe.github.io/products-engineering/code-standards/php/#multi-line-control-structures)